snowflake/1.2/modules/ROOT/pages/snowflake-connector-studio.adoc

```diff
@@ -0,0 +1,187 @@
+
+To configure the global element for Snowflake Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click *+* next to the *Connector configuration* field to access the global element configuration fields.
+. In the *General* tab, in *Required Libraries*, click *Configure* and select from the following options to install the required JDBC driver:
+* *Add recommended libraries*
+* *Use local file*
+* *Add Maven dependency*
+. In *Connection*, enter the following required information to configure authentication:
+* *Account name* +
+Account name that was provided when registering for the Snowflake account.
+* *Warehouse* +
+Name of the virtual Snowflake warehouse to use. The specified warehouse must be an existing warehouse for which the specified account has privileges.
+* *Database* +
+Snowflake database to connect to. The specified database must be an existing database for which the specified account has privileges.
+* *Schema* +
+Snowflake database schema to use. The specified schema must be an existing schema for which the specified account has privileges.
++
+Either reference a configuration file that contains ANT-style property placeholders (recommended) or enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[configure-other-fields]]
+== Configure Additional Connector Fields
+
+After you configure a global element for Snowflake Connector, configure the other required fields for the connector. The required fields vary depending on which connector operation you use.
+
+
+[[view-app-log]]
+== View the App Log
+
+To check for problems, view the app log:
+
+* If you’re running the app from Anypoint Platform, the app log output is visible in the Anypoint Studio console window.
+* If you’re running the app using Mule from the command line, the app log output is visible in your OS console.
+
+Unless the log file path is customized in the app’s log file (`log4j2.xml`), you can also view the app log in the default location `MULE_HOME/logs/<app-name>.log`. You can configure the location of the log path in the app log file `log4j2.xml`.
+
+[[configure-bouncy-castle]]
+== Configure Bouncy Castle for Encrypted Key-Pair Authentication
```

Comment:
General comment. Do you have to use the term "security provider"? It is techie sounding, especially for a cloud connector. Can you use the term "Bouncy Castle security" instead of "Bouncy Castle security provider"?